androidbrowserhelper/src/main/java/com/google/androidbrowserhelper/trusted/LauncherActivity.java

```diff
@@ -134,6 +134,26 @@ protected void onCreate(@Nullable Bundle savedInstanceState) {
             return;
         }
 
+        if (shouldLaunchImmediately()) {
+            launchTwa();
+        }
+    }
+
+    /**
+     * Signals if `LauncherActivity` should automatically launch the Trusted Web Activity on
```

Comment:
Unfortunately markdown doesn't work in javadoc comments - we've got to use a whole different annotation system.

Basically, if you want something to be code, do `{@code something}`. If you want to reference a class, do `{@link LauncherActivity}` and if you want to reference a method, do `{@link #shouldLaunchImmediately}`.

Reply:
🤦  fixed